dwave/cloud/config.py

```diff
@@ -490,29 +460,27 @@ def load_config_from_files(filenames=None):
     Examples:
         This example loads configurations from two files. One contains a default
         section with key/values that are overwritten by any profile section that
-        contains that key/value; for example, profile dw2000b in file dwave_b.conf
-        overwrites the default URL and client type, which profile dw2000a inherits
-        from the defaults section, while profile dw2000a overwrites the API token that
-        profile dw2000b inherits.
+        contains that key/value; for example, profile advantage_b in file dwave_b.conf
```

Comment:
Can we please just drop this whole example? The `load_config_from_files` is such a low-level internal function we don't want anyone ever use it :smile: 

OTOH, we don't have to modify docs significantly here, as the whole config module will be replaced in #426.